kotlin-redux/src/main/kotlin/redux/Imports.kt

```diff
@@ -0,0 +1,9 @@
+@file:JsModule("redux")
+package redux
+
+@JsName("createStore")
+internal external class Store<out S>(reducer: (S, WrapperAction) -> S, preloadedState: S) {
```

Comment:
`createStore` is actually a plain function not a constructor function, so describing it as `class` doesn't seem correct